<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mouse n Music - Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QTimeEdit" name="timeEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="displayFormat">
         <string>mm:ss:zzz</string>
        </property>
        <property name="calendarPopup">
         <bool>false</bool>
        </property>
        <property name="currentSectionIndex">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="songLengthLabel">
        <property name="text">
         <string>00:00:000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="timeBar">
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="playButton">
        <property name="text">
         <string>Play</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>x0.25</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>x0.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>x0.75</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>x1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>x1.25</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>x1.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>x2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>x4</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewSong"/>
    <addaction name="actionOpen_Map"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuPlayer">
    <property name="title">
     <string>Music</string>
    </property>
    <addaction name="actionPlay_Pause"/>
    <addaction name="actionSong_Info"/>
    <addaction name="actionPlay_Metronome"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlayer"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewSong">
   <property name="text">
    <string>New Song</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_Map">
   <property name="text">
    <string>Open Map</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionPlay_Pause">
   <property name="text">
    <string>Play/Pause</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionSong_Info">
   <property name="text">
    <string>Song Info</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
  <action name="actionPlay_Metronome">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Play Metronome</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
